doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,288 @@
+
+   .. note::
+      Since the needed drivers are part of the Service VM kernel itself, we can avoid installing the modules.
+
+5. Update Ubuntu GRUB to boot ACRN hypervisor and load ACRN Kernel Image. Append the following
+   configuration to the :file:`/etc/grub.d/40_custom`,
+
+   .. code-block:: none
+
+      menuentry 'ACRN hypervisor' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+         recordfail
+         load_video
+         gfxmode $linux_gfx_mode
+         insmod gzio
+         insmod part_msdos
+         insmod ext2
+
+         echo 'Loading ACRN hypervisor with SDC scenario ...'
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         module /boot/bzImage_sos Linux_bzImage
+      }
+
+6. Update GRUB ``sudo update-grub``.
+
+7. Shutdown the guest and relaunch using, ``virsh start ACRNSOS --console`` and select ACRN hypervisor from GRUB menu to launch Service VM running on top of ACRN.
+   This can be verified from ``dmesg`` as shown below,
+
+   .. code-block:: none
+
+      guestl1@ACRNSOS:~$ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    2.337176] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    2.368358] ACRN HVLog: Initialized hvlog module with 4 cpu
+      [    2.727905] systemd[1]: Set hostname to <ACRNSOS>.
+
+8. When shutting down, make sure to cleanly destroy the Service VM to prevent crashes in subsequent boots. This can be done using,
+
+      .. code-block:: none
+
+         virsh destory ACRNSOS # where ACRNSOS is the virsh domain name.
+
+
+Service VM Networking updates for User VM
+*****************************************
+If you are planning to enable networking for User VM(L2 guest), the following steps might be helpful.
+
+1. Edit your :file:`/etc/netplan/01-netcfg.yaml` file to add acrn-br0 as below,
+
+   .. code-block:: none
+
+      network:
+         version: 2
+         renderer: networkd
+         ethernets:
+            enp1s0:
+               dhcp4: no
+         bridges:
+            acrn-br0:
+               interfaces: [enp1s0]
+               dhcp4: true
+               dhcp6: no
+
+2. Apply the new network configuration by,
+
+      .. code-block:: none
+
+         $ cd /etc/netplan
+         $ sudo netplan generate
+         $ sudo netplan apply
+
+3. Create a tap interface (tap0) and add the tap interface as part of the acrn-br0.
+   Copy files ``misc/acrnbridge/acrn.network`` and ``misc/acrnbridge/tap0.netdev`` from the cloned ACRN repo to :file:`/usr/lib/system/network`.
+   Rename them to ``50-acrn.network`` and ``50-tap0.netdev`` respectively.
+
+4. Restart ACRNSOS guest (L1 guest) to complete the setup and start with bring-up of User VM
+
+
+Bring-up User VM (L2 Guest)
+***************************
+1. Build the device-model, using make devicemodel and copy acrn-dm to ACRNSOS guest (L1 guest) directory ``/usr/bin/acrn-dm``
```

Comment:
Give the commands (in a code-block) rather than describing the commands

Reply:
The user can build in a different system and copy it SUT. So cannot explicitly put this as command and decided to give it as instructed.